examples/error.rs

```diff
@@ -37,17 +36,32 @@ impl Handler for ErrorHandler {
 
 impl BeforeMiddleware for ErrorProducer {
     fn before(&self, _: &mut Request) -> IronResult<()> {
-        Err(IronError::new(StringError("Error".to_string()), status::BadRequest))
+        Err(IronError::new(StringError("You cannot access the resource!".to_string()), status::BadRequest))
+    }
+}
+
+// the error is simply printed on stdout and returned as is
```

Comment:
Generally speaking can you make new comments full sentences with capital letters, punctuation, etc.? It makes the examples easier to read for future new users.